EPlast/EPlast.WebApi/Controllers/LoginController.cs

```diff
@@ -146,13 +151,6 @@ public IActionResult Logout()
             _authService.SignOutAsync();
             return Ok();
         }
-
-        private readonly IAuthService _authService;
-        private readonly IJwtService _jwtService;
-        private readonly ILoggerService<LoginController> _loggerService;
-        private readonly IResources _resources;
-        private readonly IUserDatesService _userDatesService;
-
         private async Task AddEntryMembershipDate(string userId)
```

Comment:
```suggestion
        private async Task AddEntryMembershipDateAsync(string userId)
```
Could you also change this ?

Reply:
done